spec/models/comment_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
aqui vc pode testar as validações do model e associações. Se não for testar nada, acho melhor apagar o arquivo